### ISSUE TITLE
Cannot connect to Robsten Testnet by Nethereum

### ISSUE THREAD

**Issue by hessam94** [Nethereum/Nethereum#459]
Hello my friend,
I tried to connect to **Ropsten Testne**t with your API but there is a problem,
Before That i did: 
1- Create a Profile in Infura and created a project, got my ProjectId
2- Create an account in Ropsten **Through MetaMask** and got some Ether for test(13 ether now)
(I can see my account transactions in https://ropsten.etherscan.io/address/....)
3- write the solidity codes and compiled it , got the Byte code
Actually **I can deploy it on my test net on localhost and work with smart contract very well** but I cannot connect to outside addresses.
and I test it with the account number you mentioned in Docs, and MY account 

```
var web3 = new Web3("https://mainnet.infura.io");
var balance = await web3.Eth.GetBalance.SendRequestAsync("0xde0b295669a9fd93d5f28d9ec85e40f4cb697bae");
```
and
```
`   var url = "https://ropsten.infura.io/v3/ProjectId";
    var PvKey = 0x..."";`
```

The exception is InnerException = {"**The underlying connection was closed: An unexpected error occurred on a send**."}

When I **changed https to http** it get an exception for **timeout** more than 20000 millisecond. I don't think that reading a simple data needs more than 20 seconds because it is not a transaction

Can you help me please, thanks

**Comment by juanfranblanco** [comment]
https://infura.io/docs/gettingStarted/chooseaNetwork, have you got a valid "ProjectId"? and have you tried https://ropsten.infura.io although this one won't be available soon I believe.

**Comment by hessam94** [comment]
Thank you for your response,
I think it is valid because everything seems to be ok, I got my projectId, Project Secret. and I could add some Test Ethers to my account and I can see my transactions in Etherscan website. 
I do not have ping to none of these test URLs. 
cmd--> ping ropsten.infura.io gives me timeout,  
actually (pinging monorepo-production-edge- .....)
even other test networks like kovan. 
I  just have ping to infura.io
I tested it with different laptops , both of them do not work
but when I checked https://ropsten.infura.io/v3/c8bd...... on the browser it gives me 404 not found. I am confused somehow,
